common/keyboard_test.go

```diff
@@ -76,3 +76,11 @@ func TestKeyboardPress(t *testing.T) {
 		assert.Panics(t, func() { k.Press("", nil) })
 	})
 }
+
+func TestKeyDefinitionCode(t *testing.T) {
+	vu := k6test.NewVU(t)
+	k := NewKeyboard(vu.Context(), nil)
+	kd := k.keyDefinitionFromKey("a")
+
+	assert.Equal(t, "KeyA", kd.Code)
```

Comment:
Not sure, I just wanted to bake in this one behaviour. I could iterate over all the keys and ensure that the code matches the key value. Let me try.

Reply:
Yes, I think we can improve a little bit the test coverage here. It will not be bullet proof as we are the ones defining the string literals in the implementation (so we don't have a reference for which we are sure it's correct), but it might detect some typos, and we can reuse it if we add other keyboard layouts. I'm thinking on something silly like verifying the matching of the keys of [keys map](https://github.com/grafana/xk6-browser/blob/cf7b5a5840a0912f1787a71f5b69d5eacb878b6e/keyboardlayout/us.go#L261) with the code for example. Or just take the special cases as @inancgumus suggested.

Reply:
Yeah, fair point. Resolved in 39c445913ab566eeba5356a7be35d8cf3540cf0d